code-of-conduct.md

```diff
@@ -32,13 +50,21 @@ Specifically:
 * Aggression and elitism are unwelcome;
   nobody should be afraid to ask questions.
 
-We expect everyone to treat the space, bunkers and garden as well as tools
+Remember that assholes can only remain assholes
+if their entourage enables it by not saying no to their behaviour.
```

Comment:
Søren suggests:
```
Remember that discruptive behaviour often persists
because others in the environment hesitate to
establish and maintain appropriate boundaries.
```